src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -29,6 +31,75 @@ declare global {
      * A convenient reference to the file path of the Actor's profile image
      */
     get img(): this['data']['img'];
+
+    /**
+     * Provide an object which organizes all embedded Item instances by their type
+     */
+    get itemTypes(): Record<string, Item[]>;
+    /**
+     * Test whether an Actor entity is a synthetic representation of a Token (if true) or a full Entity (if false)
+     */
+    get isToken(): boolean;
+
+    /**
+     * An array of ActiveEffect instances which are present on the Actor which have a limited duration.
+     */
+    get temporaryEffects(): ActiveEffect[];
+
+    /**
+     * Return a reference to the TokenDocument which owns this Actor as a synthetic override
+     */
+    get token(): ConfiguredDocumentClassOrDefault<typeof foundry.documents.BaseToken> | null;
+    /**
+     * A convenience reference to the item type (data.type) of this Actor
+     */
+    get type(): this['data']['type'];
+    /**
+     * Apply any transformations to the Actor data which are caused by ActiveEffects.
+     */
+    applyActiveEffects(): void;
+
+    /**
+     * Retrieve an Array of active tokens which represent this Actor in the current canvas Scene.
+     * If the canvas is not currently active, or there are no linked actors, the returned Array will be empty.
+     * If the Actor is a synthetic token actor, only the exact Token which it represents will be returned.
+     *
+     * @param linked - Limit results to Tokens which are linked to the Actor. Otherwise return all Tokens even those which are not linked.
+     * @param document - Return the Document instance rather than the PlaceableObject
```

Comment:
The default values for the parameters should be documented like this:
```suggestion
     * @param linked   - Limit results to Tokens which are linked to the Actor. Otherwise return all Tokens even those which are not linked.
     *                   (default: `false`)
     * @param document - Return the Document instance rather than the PlaceableObject
     *                   (default: `false`)
```